Packages/io.chainsafe.web3-unity.web3auth/Samples~/Web3.Unity Web3Auth/Scripts/Web3AuthLogin.cs

```diff
@@ -102,17 +118,18 @@ protected override Web3Builder ConfigureWeb3Services(Web3Builder web3Builder)
                         defaultLanguage = Web3Auth.Language.en,
                         appName = "ChainSafe Gaming SDK",
                     }
-                }
+                },
+                RememberMe = rememberMe
             };
 
             if (useProvider)
             {
-                web3AuthConfig.LoginParams = new LoginParams
+                web3AuthConfig.LoginParams = new LoginParams()
```

Comment:
I wanted remember me to initially be part of the LoginParams, because that really makes sense. But the issue with that, is that LoginParams then always has to be initialized. But when LoginParams is not null, the *entire* application starts behaving so weird that I can't even explain it :D Like in WebGL it gets in sort of an infinite loop of logging in. 
So I had to make Remember Me to be part of Web3authconfig.